cloud/resources/index.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Manage Resources | Terramate Cloud
+description: Learn how Terramate helps you to manage infrastructure Resources in Terramate Cloud
+---
+
+# Resources
+
+Resources in the Terramate Cloud dashboard lets teams manage all their infrastructure assets from a single place. It monitors resources deployed with Infrastructure as Code (IaC) across multiple VCS providers, repositories, and teams, offering a complete picture of your infrastructure. The dashboard displays resource status, enabling teams to identify drifted or misconfigured assets based on deployment outcomes and drift runs.
+
+![Resources in Terramate Cloud](../assets/Resources_TMC.png)
+
+The Resources page displays all known resources across multiple repositories. Each resource highlights its essential metadata. You can sort the list—by default, resources are arranged by the time of their last update—and use filters to narrow results by repository or status. Additionally, you can search for resources by entering a search string in the name or path. Clicking on any resource opens its [Resource Details](./details.md) page.
+
+## Filter Options:
+
+- **Status**: Filter resources by their current state:
+    - `Healthy`: The resource is deployed successfully, and its current state matches the expected configuration.
+    - `Drifted`: The resource’s actual state has deviated from its declared configuration due to changes or misconfigurations.
```

Comment:
```suggestion
    - `Drifted`: The resource drifted. Which means that actual state as deployed in the cloud does not match the desired state as declared in the configuration. This can happen through manual changes, conflicting configurations, provider issues, or similar events.
```